mftf/2.3/configuration.md

```diff
@@ -56,21 +56,29 @@ Example:
 MAGENTO_ADMIN_PASSWORD=1234reTyt%$7
 ```
 
-## Advanced Configuration
+## Advanced configuration
 
-In most cases, these values are not required.
+The advanced configuration is optional.
```

Comment:
This paragraph is overly passive. 

If you are running Selenium on an external system, then you must make some additional modifications to the `.env` file. Otherwise, changing the default advanced configuration values is optional.